WalletWasabi.Gui/Controls/WalletExplorer/SendControlView.xaml

```diff
@@ -35,6 +35,8 @@
             </i:Interaction.Behaviors>
           </controls:ExtendedTextBox>
 
+          <controls:ExtendedTextBox IsVisible="{Binding IsCustomChangeAddress}" Classes="monospaceFont" Text="{Binding CustomChangeAddress}" Watermark="Custom Change Address" UseFloatingWatermark="True" />
```

Comment:
Indicate if I left this blank then change will get back to my wallet. 
Suggestions:
- Tooltip "Leave blank for default change handling"
- Watermark text "(Optional) Custom Change address"

Reply:
Yeah we were thinking of how to indicate that, but we couldn't come up with a good solution. The `(Optional)` prefix is a simple and great idea. Adding it!